### DISCUSSION TITLE
Use Cloudfront With AWS S3 For Content Delivery

### DISCUSSION THREAD

**Discussion by manishonc** [pocketbase/pocketbase#2976] [Q&A]
Hi thanks @ganigeorgiev for helping me Setup S3 .

I was wondering how I can use Cloudfront with S3 in pocketbase For Faster Video File delivery..

Or If you can suggest anything else.
For faster content delivery.

in my case video files .

**Comment by ganigeorgiev** [comment]
At the moment there is no option to specify a CDN address (see https://github.com/pocketbase/pocketbase/issues/28#issuecomment-1492516269).

Alternatively, if the files are public you can use directly the public CDN URL. The object keys are in the format `{collectionId}/{recordId}/{filename}`, for example:
  ```
  https://yourcdn.com/yourCollectionId/yourRecordId/yourFile.mp4
  ```

**Reply by manishonc** [reply]
Thanks I this method is working. I testing the same.
